### ISSUE TITLE
update for application 4.0.0 cookbook

### ISSUE THREAD

**Issue by wish-sge** [poise/application_php#14]
Hi,

Is there any update possible for compatibility to a application 4.0.0 cookbook? I think the only thing that has changed with application 4.0.0 is that it removes compatability with Chef 10.

Thanks.

**Comment by arangamani** [comment]
Just submitted a pull request here #16. It seems to work fine for me with 4.0 version of application cookbook.

**Comment by jbrook** [comment]
Yes please. This completely broke my setup with Berkshelf. I had fixed the application cookbook version on v4.1.4 and then later added application_php without a specific version and berkshelf could not resolve the dependency because of the pessimistic constraint on v3.

**Comment by arangamani** [comment]
@coderanger ^^. Thanks in advance though :smile:
